src/components/Projects/ProjectsList.jsx

```diff
@@ -16,28 +16,37 @@ function ProjectsItem(props) {
 }
 
 export function ProjectsList() {
-  const result = search();
-  if (result.length !== 0)
-    return (
-      <div className="container">
-        <section className="projects-list">
-          {result.map((item) => {
-            return (
-              <ProjectsItem
-                src={item.src}
-                title={item.title}
-                text={item.text}
-                key={item.title}
-              />
-            );
-          })}
-        </section>
-      </div>
-    );
-  else
+  const result = search().result;
+  const isLoaded = search().isLoaded;
```

Comment:
Here should be something like 
```suggestion
  const isLoaded = useSelector(projectsSelector).isLoaded;
```